extensions/parquet/compression/build.gradle

```diff
@@ -12,6 +12,8 @@ dependencies {
     implementation project(':Configuration')
 
     implementation depCommonsIo
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.1'
+    // TODO Is there an alternate to specifying version, the default doesn't work
```

Comment:
As of today (if you are up-to-date with main), you should be able to do `implementation libs.jackson.databind`. But I commented elsewhere, we shouldn't depend on this here IMO.

Reply:
Replied [here](https://github.com/deephaven/deephaven-core/pull/5712#discussion_r1669118933).